website/examples/e-commerce/Theme.css

```diff
@@ -0,0 +1,535 @@
+.ais-RatingMenu-item,
+.ais-RefinementList-item {
+  padding-bottom: 1rem;
+}
+
+.ais-Breadcrumb-item--selected,
+.ais-HierarchicalMenu-item--selected,
+.ais-Menu-item--selected {
+  font-weight: bold;
+}
+
+.ais-RatingMenu-starIcon--full {
+  fill: #e2a400;
+}
+
+.ais-RatingMenu-starIcon--empty {
+  fill: rgba(0, 0, 0, 0.08);
+}
+
+/* Panel */
+
+.ais-Panel--collapsible {
+  position: relative;
+}
+
+.ais-Panel--collapsed .ais-Panel-body,
+.ais-Panel--collapsed .ais-Panel-footer {
+  display: none;
+}
+
+.ais-Panel-collapseButton {
+  background: none;
+  border: none;
+  cursor: pointer;
+  outline: none;
+  padding: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+}
+
+.ais-Panel-header {
+  border: none;
+  color: #21243d;
+  font-size: 0.678rem;
+  font-weight: 600;
+  letter-spacing: 0.08rem;
+  line-height: 1.6;
+  padding-bottom: 1rem;
+  text-transform: uppercase;
+}
+
+/* Search box */
+
+.ais-SearchBox-input {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  background-color: rgba(65, 66, 71, 0.06);
+  border: none;
+  border-radius: 3px;
+  color: rgba(33, 36, 61, 0.8);
+  font-family: inherit;
+  min-height: 54px;
+  outline: none;
+  padding-left: 56px;
+  width: 100%;
+}
+
+.ais-SearchBox-input::placeholder {
+  color: rgba(33, 36, 61, 0.5);
+  opacity: 1; /* Firefox */
+}
+
+.ais-SearchBox-input:-ms-input-placeholder {
+  color: rgba(33, 36, 61, 0.5);
+}
+
+.ais-SearchBox-input::-ms-input-placeholder {
+  color: rgba(33, 36, 61, 0.5);
+}
+
+.ais-SearchBox-reset,
+.ais-SearchBox-loadingIndicator,
+.ais-SearchBox-submit {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.ais-SearchBox-submit {
+  left: 26px;
+}
+
+.ais-SearchBox-reset,
+.ais-SearchBox-loadingIndicator {
+  right: 26px;
+}
+
+.ais-SearchBox-form {
+  position: relative;
+}
+
+.ais-SearchBox-loadingIndicator,
+.ais-SearchBox-reset,
+.ais-SearchBox-submit {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+/* SFFV search box */
+
+.ais-RefinementList .ais-SearchBox-input {
+  border-radius: 3px;
+  color: rgba(33, 36, 61, 0.8);
+  font-size: 0.8rem;
+  min-height: 40px;
+  padding-left: 36px;
+}
+
+.ais-RefinementList .ais-SearchBox-submit {
+  left: 1rem;
+}
+
+.ais-RefinementList .ais-SearchBox-loadingIndicator,
+.ais-RefinementList .ais-SearchBox-reset {
+  right: 1rem;
+}
+
+.ais-RefinementList .ais-SearchBox-form {
+  margin-bottom: 1rem;
+}
+
+/* Menus */
+
+.ais-HierarchicalMenu-link,
+.ais-RatingMenu-link,
+.ais-RefinementList-label {
+  align-items: center;
+  display: flex;
+}
+
+.ais-RefinementList-checkbox {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  background-color: rgba(65, 66, 71, 0.08);
+  border-radius: 2px;
+  height: 1rem;
+  margin: 0;
+  margin-right: 1rem;
+  outline: none;
```

Comment:
I didin't realise that we removed the outline. Indeed it looks nicer without it, but it's a nightmare to navigate with the keyboard without this indication. We should keep it, but not the default color.

Reply:
That's something that I mentioned in the first PR in InstantSearch.js. There's a lot of accessibility improvements to do, but they should be shared across all the e-commerce demos so let's not treat them as blockers.

I'll ask Nico if he can include this in the next design additions he plans to do.

Reply:
Fair enough I wasn’t sure that we already talked about it.